src/pages/ReimbursementAccount/reimbursementAccountPropTypes.js

```diff
@@ -28,7 +28,7 @@ export default PropTypes.shape({
     error: PropTypes.string,
 
     /** Which field needs attention? */
-    errorFields: PropTypes.objectOf(PropTypes.bool),
+    errorFields: PropTypes.objectOf(PropTypes.oneOfType([PropTypes.bool, PropTypes.array])),
```

Comment:
Confirmed that error types show up correctly for both types: 

**Boolean**
<img width="1416" alt="Screen Shot 2022-10-10 at 17 00 31" src="https://user-images.githubusercontent.com/16747822/194896279-9a8fca4d-0cfb-41f1-bfea-e487dff491c2.png">


**Array**

<img width="1208" alt="Screen Shot 2022-10-10 at 16 43 32" src="https://user-images.githubusercontent.com/16747822/194895526-08659b75-23b5-4266-a156-aa9704b8e7f0.png">